op-node/rollup/types.go

```diff
@@ -55,6 +55,7 @@ type Config struct {
 	Genesis Genesis `json:"genesis"`
 	// Seconds per L2 block
 	BlockTime uint64 `json:"block_time"`
+
```

Comment:
it doesn't hurt, but changing this file is unrelated to this PR, maybe remove this diff.